andela/merry-ah-frontend
# #162727439 Implement List bookmark arts

#### What does this PR do?
List all bookmarked arts
#### Description of Task to be completed?
- Get all bookmarked arts by the user 
- Write tests
#### How should this be manually tested?
- Run `npm i`
- Start the server `npm run start:dev`
- Navigate to `localhost:5050/api/v1/bookmark` to see bookmark arts
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#162727439](https://www.pivotaltracker.com/story/show/162727439)
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2019-02-27 at 2 04 02 pm" src="https://user-images.githubusercontent.com/6943256/53492207-9887fe00-3a98-11e9-8417-e2f44fb4e6cf.png">
